backend/database/repositories/person.go

```diff
@@ -11,11 +11,12 @@ import (
 // Implements IPersonRepository
 type personRepository struct {
 	embeddedContext
+	FrontEndID int
```

Comment:
nitpicks:
 - Swap order of fields
 - FrontEndID should not be exported (it should start with a lowercase letter)